lib/bracket-matcher.coffee

```diff
@@ -145,7 +151,7 @@ class BracketMatcher
     bracketsRemoved
 
   wrapSelectionInBrackets: (bracket) ->
-    return false unless atom.config.get('bracket-matcher.wrapSelectionsInBrackets')
+    return false unless @getScopedSetting('bracket-matcher.wrapSelectionsInBrackets')
 
     if bracket is '#'
```

Comment:
As discussed on https://github.com/atom/bracket-matcher/pull/223/files/26d598d607bdfb899932b45f4b867820a53ba552#r58013091, I actually believe it's fine to use `getLastCursor().getScopeDescriptor()`, because at least we will get a correct behavior in the case of single edits, which are the most common when writing code. :+1: